File: Pascal/fizzbuzz1.pas
program FizzBuzz1;
const
	limit = 100;
var
	i: integer;
begin
	for i := 1 to limit do
	begin
		if i mod 15 = 0 then
			write('FizzBuzz')
		else if i mod 3 = 0 then
			write('Fizz')
		else if i mod 5 = 0 then
			write('Buzz')
		else
			write(i);
		if i < limit then write(', ')
	end;
	writeln()
end.

File: Nim/fizzbuzz1.nim
const limit = 100

for i in 1..limit:
  if i mod 15 == 0:
    stdout.write "FizzBuzz"
  elif i mod 3 == 0:
    stdout.write "Fizz"
  elif i mod 5 == 0:
    stdout.write "Buzz"
  else:
    stdout.write i
  if i < limit: stdout.write ", "

stdout.write '\n'
